source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst

```diff
@@ -222,7 +222,7 @@ Now, under the switch node we add the following:
     .. code-block:: console
 
         root@<machine>:~# cat /proc/interrupts | grep "MCUGPIO"
-        262:          0          0          0          0      GPIO  16 Edge    -davinci_gpio  MCUGPIO
+        273:          0          0          0          0      GPIO  4 Edge    -davinci_gpio  MCUGPIO
```

Comment:
Is the `4` here some specific pin? If it is, then this section is not machine agnostic and should actually say what machine it is in the code-block.